xarray/core/rolling_exp.py

```diff
@@ -0,0 +1,70 @@
+
+import numpy as np
+from pandas.core.window import _get_center_of_mass
```

Comment:
We're going to want to vendor this private pandas function in xarray -- we don't want to be dependent on private APIs in pandas.